posthog/models/funnel.py

```diff
@@ -42,7 +43,7 @@ def _gen_lateral_bodies(self, team_id: int, filter: Filter):
                     **{filter_key: step.id},
                     team_id=team_id,
                     **({"distinct_id": "1234321"} if index > 0 else {}),
-                    **({"timestamp__gte": "2000-01-01"} if index > 0 else {}),
+                    **({"timestamp__gte": timezone.now().replace(year=2000, month=1, day=1, hour=0, minute=0, second=0, microsecond=0)} if index > 0 else {}),
```

Comment:
Couldn't you just do "2000-01-01T00:00:00Z"?

Reply:
Yep that's much better.